### PR TITLE
Removed __unicode__() in mongonaut_tags.py. Added a unit test

### PR DESCRIPTION
**Pull request by lchsk**
Solution for the problem mentioned in https://github.com/pydanny/django-mongonaut/issues/79 with a unit test

### CODE REVIEW

File: examples/blog/articles/models.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ class NewUser(User):
     new_field = StringField()
 
     def __unicode__(self):
-        return self.email
+        return unicode(self.first_name)
```
Comment (garrypolley) [inline examples/blog/articles/models.py]: Why is this changed to `first_name`? It no longer matches the `User` class above. I'd prefer this continue to be the email. 

Also, why is this getting caste to unicode?

File: tests/view_tests.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#coding: utf-8
+# -*- coding: utf-8 -*-
```
Comment (garrypolley) [inline tests/view_tests.py]: If this change is going to be made it should probably be made to all the files in the project rather than a one off basis. Personally I don't think it really matters, so I'm fine with no change too.

**Comment by lchsk** [comment]
I've used `first_name` so it would make more sense to put non-ascii characters there. But I can change it back to `email`. I'll send a pull request afterwards.

**Comment by garrypolley** [comment]
@lchsk  No big deal. I just noticed that was the example application. No need to change it.

**Comment by garrypolley** [comment]
I'm cool with these changes.  :+1:   I'll give some others a day or so to look these over and then I'll merge it in. 

Thanks for the pull request.

**Comment by pydanny** [comment]
I think stronger support of unicode is always a good thing. I agree with @garrypolley that we shouldn't switch from email.

**Comment by garrypolley** [comment]
@lchsk Are you currently using what is in this pull request? It's not had much activity for months. 

If you need it I can hit the merge button, but I'd like to see the email used as mentioned above.

**Comment by lchsk** [comment]
Hi
I totally forgot about this (I'm embarrassed enough :(). But yes, I use version from the pull request. I removed unnecessary changes I had made previously. I would be great if you could merge it. Cheers

**Comment by garrypolley** [comment]
Never feel embarrassed, thanks for the contribution! We appreciate any help we can get on open source projects.
